exp/api/v1alpha3/azuremanagedcontrolplane_webhook_test.go

```diff
@@ -67,7 +67,7 @@ func TestDefaultingWebhook(t *testing.T) {
 	g.Expect(*amcp.Spec.LoadBalancerSKU).To(Equal(lbSKU))
 	g.Expect(*amcp.Spec.NetworkPolicy).To(Equal(netPol))
 	g.Expect(amcp.Spec.Version).To(Equal("v9.99.99"))
-	g.Expect(*&amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
+	g.Expect(&amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
```

Comment:
Perhaps, this will do the trick.
```suggestion
	g.Expect(amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
```

Reply:
Thanks that did the trick (at least, locally)! :)